cincinnati/src/plugins/internal/graph_builder/release_scrape_dockerv2/plugin.rs

```diff
@@ -134,6 +137,10 @@ impl ReleaseScrapeDockerv2Plugin {
 #[async_trait]
 impl InternalPlugin for ReleaseScrapeDockerv2Plugin {
     async fn run_internal(self: &Self, io: InternalIO) -> Fallible<InternalIO> {
+        get_tracer()
+            .get_active_span()
+            .update_name("registry-scrape".to_string());
```

Comment:
Use the predefined constant instead:
```suggestion
            .update_name(Self::PLUGIN_NAME.to_string());
```